app/helpers/react_components/mentoring/discussion_post_list.rb

```diff
@@ -0,0 +1,18 @@
+module ReactComponents
```

Comment:
In the end, this probably won't exist as we're definitely rendering this component as a child of a bigger mentoring component. I just added this now as I want to get end to end tests for the websockets notification.